UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -184,6 +184,14 @@ <h2>Publisher metadata ecosystem</h2>
                             Finally from the "Metadata Processing" layer both the Display Techniques for EPUB and ONIX flow into the "Resulting Statements" layer "User Experience Guide for Displaying Accessibility Metadata"</p>
                     </details>
 
+<h2>Legal Considerations</h2>
+
+<p> in some jurisdictions  publishers may have an exemption from the provision of accessible publications, including the provision of accessibility metadata. Some examples of this are when the publisher claims a disproportionate burden  exemption or when it is a micro-enterprise (e.g., enterprises employing fewer than 10 people and with annual
```

Comment:
This criteria for micro is for the EAA, may be different in other legislation. I would prefer not to include any example, eventually a note pointing to the EAA and later on other legislation.